test/selenium/sessions_ui_test.js

```diff
@@ -255,6 +288,34 @@ describe('sessions ui', function() {
                            5000);
       });
     });
-  });
 
+    describe('tests changing views', function(){
+
+        before(function() {
+          driver.navigate().refresh();
+          signInPage = new SignInPage(driver);
+
+        });
+        beforeEach(function(){
+          return driver.wait(webdriver.until.titleIs('FoxBox'), 5000).then(
```

Comment:
*No need to change this line in the current PR*

This wait is done at some other places, we should think of extracting it in a page class. This is better done is a follow up PR